packages/client/src/i18n/messages/views/config.ts

```diff
@@ -67,6 +71,28 @@ const messagesToDefine: IConfigMessages = {
     defaultMessage: 'Unable to make change. Please try again',
     description: 'Error message for application name change'
   },
+  govtLogoChangeMessage: {
+    id: 'config.application.govtLogoChangeMessage',
+    defaultMessage:
+      'Upload the Government logo to be used on the login and form decalation. Note certificate logo is uploaded as part of the certificate template.',
+    description:
```

Comment:
**defaultMessage** and **description** are the same here. Maybe you forgot to update.